dev/format_python_code.sh

```diff
@@ -15,29 +16,14 @@ if ! ${ISORT} --version >/dev/null 2>&1; then
     exit 1
 fi
 
-# The commands must be run on each module directory separately,
-# otherwise tools confuse the "current" module
-for paths in \
-    "cvat-sdk" \
-    "cvat-cli" \
-    "tests/python/" \
-    "cvat/apps/quality_control" \
-    "cvat/apps/analytics_report" \
-    "cvat/apps/engine/lazy_list.py" \
-    "cvat/apps/engine/background.py" \
-    "cvat/apps/engine/frame_provider.py" \
-    "cvat/apps/engine/cache.py" \
-    "cvat/apps/engine/default_settings.py" \
-    "cvat/apps/engine/field_validation.py" \
-    "cvat/apps/engine/model_utils.py" \
-    "cvat/apps/engine/task_validation.py" \
-    "cvat/apps/dataset_manager/cron.py" \
-    "cvat/apps/dataset_manager/tests/test_annotation.py" \
-    "cvat/apps/dataset_manager/tests/utils.py" \
-    "cvat/apps/events/signals.py" \
-    "cvat/apps/engine/management/commands/syncperiodicjobs.py" \
-    "cvat/apps/dataset_manager/management/commands/cleanuplegacyexportcache.py" \
-    ; do
-    ${BLACK} -- ${paths}
-    ${ISORT} -- ${paths}
-done
+cd -- "${REPO_ROOT}"
+${BLACK} .
+
+# isort is slow at skipping large Git-ignored directories when invoked as `isort .`,
+# so pass it an explicit list of files instead.
+
+# isort will only skip files listed in the config file corresponding to the first file
+# on the command line, which would normally be cvat-cli/pyproject.toml. Force the first
+# file to be manage.py, so that it uses `pyproject.toml` instead.
+git ls-files -z --exclude-standard --deduplicate --cached --others '*.py' \
+    | xargs -0 ${ISORT} --resolve-all-configs --check --filter-files manage.py
```

Comment:
```suggestion
    | xargs -0 ${ISORT} --resolve-all-configs --filter-files manage.py
```
Why only check?

Reply:
Oh, oops. Sorry about that - I forgot to remove it after testing.